tests/integration/replication-rdbchannel.tcl

```diff
@@ -783,7 +776,7 @@ start_server {tags {"repl external:skip"}} {
                 [s 0 sync_full] == 2 &&
                 [s 0 sync_partial_ok] == 0
             } else {
-                fail "sync was not successful [s 0 sync_full] [s 0 sync_partial_ok]"
```

Comment:
why are all these changes here?

Reply:
Please see after the last commit. Just pushed a few minor test fixes as part of this PR (not to create another PR for these minor fixes). A comment fix, deletion of a noop operation in the test, fix for time sensitive flaky test.

Reply:
i see the test race being mentioned in the top comment, but i didn't see the `fail` messages changes or cleanups being mentioned so i didn't understand. i don't mind keeping the cleanup, just wanted to know what it was about.

Reply:
okay, test changes are either cleanup or just to fix flaky ones. I ran some more dailys last a few days and added couple of flaky test fixes more.